reference/halyard/component-sizing.md

```diff
@@ -15,21 +15,26 @@ The following Kubernetes settings can be tweaked within custom sizing.
 
 ### CPU & Memory requests/limits
 
-Container requests and limits for cpu and memory can be specified in the customSizing section of the Halyard Configuration. Note that if you specify the service name, these sizings will also be applied to any sidecars running alongside the primary container. To apply sizing to only a specfic container and not the sidecars, specify the container name instead of the service name. Please see the example for details.
+Container requests and limits for cpu and memory can be specified in the `deploymentEnvironment.customSizing` section of the halconfig file. 
 
-```
+There are two ways to specify requests and limits
```

Comment:
Even though you say there are "two ways," use bullets instead of numbers, because they're not meant to be done in order.